lib/fabrik/Agent.js

```diff
@@ -96,13 +142,9 @@ class Agent extends HttpClient {
       .any(_.map(ips, ip => this
         .getInfo(ip)
         .then(info => {
-          logger.debug(`checking ${feature} in ${ip} agent info - ${info}`);
-          const api_version = _.get(info, 'api_version', '1.0');
-          if (compareVersions(api_version, '1.1') < 0) {
-            if (_.includes(['credentials'], feature)) {
-              return ip;
-            }
-          } else if (compareVersions(api_version, '1.2') < 0) {
+          logger.debug(`checking ${feature} in ${ip} agent info`, info);
+          const api_version = _.get(info, 'api_version', '1');
```

Comment:
if we make api_version to 1 from 1.0 here , will the code be backward compatible ? here - https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/153/files#diff-3583229b85457b2837d2485caf38916dR54

Reply:
Yes, as in next line I check for both fraction version (only 1.x) and integer version.
 **`if (compareVersions(api_version, '1.3') < 0 || Number.isInteger(Number(`${api_version}`))) {`**

Reply:
Ok , thanks for clarifying.